schemas/host.yml

```diff
@@ -79,3 +79,14 @@
       example: "x86_64"
       description: >
         Operating system architecture.
+
+    - name: domain
+      level: extended
+      type: keyword
+      short: Name of the directory the group is a member of.
+      description: >
+        Name of the directory the group is a member of.
```

Comment:
I would think this would read like:

"Name of the domain of which the host is a member. For example, on Windows this could be the host's Active Directory domain or NetBIOS domain name. For Linux this could be the domain of the host's LDAP provider."

Reply:
Fixed!